File: SustitucionMOA/src/app/common/models/ordenes-de-carga-fason/editarOrdenDeCargaFasonRequest.ts
import { UnidadTransporteCarga } from "../ordenes-de-carga-common/unidadTransporteCarga";
import { OrdenDeCargaFasonDto } from "./ordenDeCargaFasonDto";

export class EditarOrdenDeCargaFasonRequest extends OrdenDeCargaFasonDto {

    public UnidadTransporte: UnidadTransporteCarga;
}

File: SustitucionMOAModel/Dto/OrdenDeCargaFason/EditarOrdenDeCargaFasonRequest.cs
using SustitucionMOAModel.Dto.OrdenDeCargaCommon;

namespace SustitucionMOAModel.Dto.OrdenDeCargaFason
{
    public class EditarOrdenDeCargaFasonRequest : OrdenDeCargaFasonRequest
    {
        public UnidadTransporteCarga UnidadTransporte { get; set; }
    }
}
